package.json

```diff
@@ -17,19 +14,19 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/sitebase/react-avatar.git"
+    "url": "https://github.com/QuickBase/react-avatar.git"
   },
   "author": "Wim Mostmans <wim@sitebase.be>",
   "bugs": {
-    "url": "https://github.com/sitebase/react-avatar/issues"
+    "url": "https://github.com/QuickBase/react-avatar/issues"
```

Comment:
I vote we leave bugs and homepage as the original. We are only a fork. We are not planning on supporting this any longer than we need to.